### PR TITLE
fixes for infuse and pull from mobile

### PR DESCRIPTION
**Pull request by inexorableAce**
infuse before:
image

infuse after:
image

pull from before:
image

pull from after:
image

### CODE REVIEW

**Comment by Wh1rledPeas** [comment]
Thanks for looking into this Ace. From the looks of your screenshots, it appears as though I should have provided more information about the layout issues. It doesn't look like you have enough items in the list to replicate the problem. Here is a video that illustrates what happens when the list starts to scroll.

[!Screenshot_20191104-201556](https://youtu.be/qafROGs-7U8)

File: src/app/infuse/InfusionFinder.tsx [hunk 1]
```diff
@@ -257,7 +257,7 @@ class InfusionFinder extends React.Component<Props, State> {
                   </div>
                 ))}
               </div>
-              <div className="itemGrid">
```
Comment (bhollis) [inline src/app/infuse/InfusionFinder.tsx]: You can delete this class from `InfusionFinder.scss`

File: src/app/inventory/StoreBucket.scss [hunk 2]
```diff
@@ -13,6 +13,11 @@
   align-content: flex-start;
   padding: 4px 0 calc(var(--item-margin) + 4px) 0;
 
+  &:empty {
```
Comment (bhollis) [inline src/app/inventory/StoreBucket.scss]: Oh neat, I didn't know about this!

File: src/app/infuse/InfusionFinder.scss [hunk 3]
```diff
@@ -8,10 +8,6 @@
   max-width: 800px;
   margin: 0 auto;
 
-  @include phone-portrait {
-    --item-margin: 6px;
```
Comment (bhollis) [inline src/app/infuse/InfusionFinder.scss]: This was originally meant to help squeeze an extra item into the grid, not sure if it's needed anymore.

**Comment by inexorableAce** [comment]
@Wh1rledPeas yup this should fix that as well. It's in beta and has been patched to prod give it a go and let me know. But this should fix all the layout stuff with those 2 sheets, that overflow scroll thing included.

**Comment by Wh1rledPeas** [comment]
@inexorableAce It's working great in Production right now. Thanks for fixing this. Made pulling anything local, very difficult without the ability to filter. 😳

**Pull request merged**